Untitled.ipynb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This looks like an empty notebook.  You can remove it with the following command `git rm Untitle.ipynb`.